internal/signaling/peer.go

```diff
@@ -192,7 +192,7 @@ func (p *Peer) HandleHelloPacket(ctx context.Context, packet HelloPacket) error
 	if packet.ID != "" && packet.Secret != "" {
 		logger.Info("peer reconnecting", zap.String("game", packet.Game), zap.String("peer", packet.ID))
 		var err error
-		hasReconnected, reconnectingLobbies, err = p.retrievedIDCallback(ctx, packet.ID, packet.Secret, packet.Game)
+		hasReconnected, reconnectingLobbies, err = p.retrievedIDCallback(ctx, packet.ID, packet.Secret)
```

Comment:
Do we still check if the game is the same? It could lead to weird cases if the gameID suddenly switches (because a malicious reconnect)

Reply:
I have brought this relation back now.